data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+########################################################################
+
+#Investigate a theorised smuggling route and rescue Hai. You have to do at least three of these missions.
+
+#NOTE: The Hai Leaks Response 6 and 7 missions will break if any of these missions have Hai-home as a destination.
+
+mission "Hai Rescue: Yeertle Family"
+	landing
+	name "Return the Yeertle family"
+	description "Transport the Yeertle family to <destination>. Payment is <payment>."
+	source Prime
+	destination Icelake
+	passengers 5
+	repeat 3
+	to offer
+		"combat rating" > 50
+		has "First Contact: Hai: offered"
+		not "event: hai-human treaty signed"
+		"Hai Rescue: Yeertle Family (offer count)" < 3
+		not "Hai Rescue: Yeertle Family: offered"
+	on offer
+		"Hai Rescue: Yeertle Family (offer count)" ++
+		conversation
+			`You look ahead a few dozen meters and you see a short man with a strange necklace. As he walks past a power transfer unit, the man shimmers and you see faintly-visible, squirrel-like, features. It seems the necklace is a projector of a human hologram concealing a Hai. He is sticking to the shadows, trying to avoid being seen.`
+			`	Do you want to talk to him?`
+			choice
+				`	(Talk to him.)`
+				`	(Come back later.)`
+					defer
+			`	The man's expression doesn't change much as you approach. His holoprojector may not be sophisticated enough to translate Hai facial expressions. You can see exposed wires on the device; this is probably a makeshift projector he produced with limited resources. A well-made version might keep Hai hidden in human space for a long time.`
+			`	From the shaking voice, you can tell he's frightened.`
+			`	"C-can I help you?"`
+			choice
+				`	"You're a Hai. Why are you this far into human space?"`
+					goto far
+				`	"Do you need a ride home to Hai space?"`
+					goto "ride offer"
+			label far
+			`	He pauses for a second, perhaps pondering if he can trust you with the answer.`
+			`	"I'm a holographer from Icelake. I chartered a flight from Icelake to Allhome with my family. They would have gone on vacation while I would have been at a conference. Instead, the human merchants locked us in a cell and took us here."`
+			`	"My daughter is very small for her age, and was able to sneak through a vent to unlock our cell. The ship was under-crewed, so we overpowered them, but they had already landed. The ship was locked down; we could not take off. Since then, we've been hiding near here."`
```

Comment:
```suggestion
			`	"My daughter is very small for her age, and was able to sneak through a vent to unlock our cell. The ship was under-crewed, so we overpowered them. Unfortunately, they had already landed, and the ship is locked down. I don't know how to disable it so we can take off. We've been hiding near here since then."`
```
Didn't really like the structuring here. Tried to change it so it flowed a little better.

Reply:
I think I am going to attempt a broader re-write of this paragraph.